drivers/virtualbox/virtualbox.go

```diff
@@ -533,7 +576,7 @@ func (d *Driver) diskPath() string {
 func (d *Driver) generateDiskImage(size int) error {
 	log.Debugf("Creating %d MB hard disk image...", size)
 
-	magicString := "boot2docker, please format-me"
+	magicString := "data"
 
 	buf := new(bytes.Buffer)
 	tw := tar.NewWriter(buf)
```

Comment:
Couldn't we skip the `tar` completely now and just initialize a blank disk instead?